eisenhower-matrix/src/main/webapp/style.css

```diff
@@ -46,41 +44,38 @@ h1 {
   font-size: 19px;
   padding: 10px;
   border-radius: 5px;
-}
-
-#inputHolder {
-  margin-top: 2%;
-  font-size: 20px;
+  position: relative;
+  top: -20px;
 }
 
 .required:before { 
   content:"*"; 
   color: red;
-  font-size: medium;
 }
 
-input {
-  position: relative;
-  bottom: 2px;
-  padding: 3px;
-  width: 97%;
+.hide {
+  display: none;
 }
 
-select {
-  width: 100%;
+#submit {
+  bottom: -90px;
 }
 
-#submitButton {
-  bottom: -90px;
+#submit:hover, #login:hover, #logout:hover {
+  background-color: #daaeae;
+  color: white;
 }
 
-.hide {
-    display: none;
+#submit {
+  position: relative;
+  left: 50%;
+  transform: translate(-50%, -50%);
```

Comment:
another example would be here with transform; I think using flexbox & centering this button would be more responsive but it's not critical, just some cool CSS stuff I'm trying to learn more about